modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,355 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="creating-automated-etcd-backups_{context}"]
+= Creating automated etcd backups
+
+The automated backup feature for etcd supports both recurring and single backups. Recurring backups create a cron job that starts a single backup each time it triggers.
+
+:FeatureName: Automating etcd backups
```

Comment:
Technically, yes, but unfortunately, the technology preview snippet has a hard-coded "is" in it ("{FeatureName} is a Technology Preview feature only").

In order to avoid plural/singular disagreement, this was a compromise. :slightly_smiling_face: